lib/uktest/Config.uk

```diff
@@ -1,8 +1,8 @@
 menuconfig LIBUKTEST
 	bool "uktest: Unikraft Test Suite"
 	default n
-	select LIBNOLIBC if !HAVE_LIBC
-	select LIBUKDEBUG
+	depends on (HAVE_LIBC || LIBNOLIBC)
+	depends on LIBUKDEBUG
```

Comment:
This change is probably okay given that a libc and ukdebug is typically always part of a build. But I see it as a compromise to resolve the circular dependency issue.